style.css

```diff
@@ -110,10 +114,280 @@
   height: 340px;
   position: absolute;
   left: 62.67%;
-  top: 50.1%;
+  top: 4.4%;
   bottom: 1.8%;
 }
 
 #container-2 {
   list-style: none;
 }
+
+.works {
+  margin-top: 72px;
+  width: 375px;
+  background-color: #fff;
+}
+
+#header-2 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 327px;
+  height: 83px;
+  margin-top: 60px;
+  font-size: 40px;
+  font-family: 'Crete Round', sans-serif, 'Times New Roman', Times, serif;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 52px;
+  padding-top: 70px;
+  padding-left: 18px;
+}
+
+#line {
+  width: 327px;
+  margin-bottom: 50px;
+}
+
+#gymfit-container {
+  width: 327px;
+  height: 237px;
+}
+
+#gymfit-img {
+  width: 327px;
+  height: 237px;
+  padding-left: 24px;
+}
+
+#header-3 {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', sans-serif, 'Times New Roman', Times, serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 32px;
+  padding-left: 24px;
+}
+
+#stories-para {
+  width: 327px;
+  height: 147px;
+  font-family: 'Inter', sans-serif, 'Times New Roman', Times, serif;
+  font-size: 16px;
+  font-weight: 400px;
+  line-height: 24px;
+  font-style: normal;
+  padding-left: 24px;
+}
+
+.language {
+  width: 297px;
+  height: 40px;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  padding-left: 24px;
+  gap: 12px;
+  text-align: center;
+}
+
+.btn button {
+  width: auto;
+  font-family: 'Inter', sans-serif, 'Times New Roman', Times, serif;
+  font-weight: 500;
+  font-style: normal;
+  line-height: 20px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+  height: 32px;
+}
+
+#btn-container {
+  width: 124px;
+  height: 48px;
+  padding-left: 24px;
+  margin: 0;
+}
```

Comment:
- [ ] Good job so far 👏 👏 Your design looks great. In addition to what you have done, I would suggest you add different styles based on various user interactions like `hover`, `active`, or `pressed` to all your buttons. This enhances user experience and makes your UI awesome 😉 You have been provided with the design guideline 👇 
![image](https://user-images.githubusercontent.com/90222110/192111862-3f3c1c8f-d4a3-460e-afea-575ff231231e.png)

Reply:
I added the interactive buttons as told.